### DISCUSSION TITLE
breadcrumbs doubles content

### DISCUSSION THREAD

**Discussion by kreincke** [bootscore/bootscore#368] [General]
Please let me again ask two questions. If you agree with my viewpoint I will create the respective issues / prs. Both questions concern the breadcrumbs in the bootscore standard version:

1.  In functions.php bootscore-main delivers the function 'the_breadcrumb'. It starts with one(!) #nav>, followed by one(!) #ol>and some opened and closed #li>-#/li> pairs. After having got the last breadcrumb the list is closed by one #/ol>. but the nav-area is closed by two(!) #/nav>-tags.  What's the reason for the two closing #/nav> tags? Clould it be that one of it is superfluous?

2. In single.php the function 'the_breadcrumb' is called as part of the hook and delivers a nice home button, all categories the post has been classified as and the title of the post inside of a nice rounded box. Then, inside the semantic #header> area as part of #main>, the function 'bootscore_category_badge()' delivers a nice list of all categories the post has been classified with. And directly after that list, 'the_title()' inserts the title of the post. Is there a 'serious' reason why you integrate the same information on the single-page twice? Or do you do that only for the purpose to demo how we can 'pimp' the pages?

many thanks for your patience
KR

**Comment by crftwrk** [comment]
First question: @sean-au?

Second question: What do you mean with content twice? Of course, breadcrumb lists category and page title as well. That's what a breadcrumb should do. But breadcrumb does not replace the category badges or the `<h1>` page title, it is an additional component like you see on every news site.

**Comment by kreincke** [comment]
... here is a screen shot of one post of my redesign material:

![doubles](https://user-images.githubusercontent.com/1419632/213532517-fe72b113-1e8b-41af-9fe4-d55d288cccba.png)


 the lines should indicate that the same information is presented twice - directly under each other. From the viewpoint of a reader who should get without any irritations what he looks for this is irritating. I've never seen a blog which goes this way (and even you yourself also refrain from displaying both https://bootscore.me/documentation/ )

May be there are reasons to go this way which I want to understand before I modify your work 

best regards kr

**Comment by crftwrk** [comment]
Yes, this is exactly what a breadcrumb should do: showing the path to your post. Category and title are part of your url, that‘s why breadcrumb shows them too. But again, breadcrumb is the path and does not replace title or anything else. Check some news pages, breadcrumb, category and title are everywhere the same.

![9277D698-3651-410D-8590-13535A9E9522](https://user-images.githubusercontent.com/51531217/213546229-91ba4b88-db17-4c09-a651-0751d353b965.jpeg)

But of course, you can delete or reorder breadcrumb, badges or title if you want ;-)

**Comment by kreincke** [comment]
ok. Many thanks for this background information. Very Good example!! I will incorporate it - maybe with a modified design which indicates the difference a bit more.

best regards KR

**Comment by kreincke** [comment]
I have to ask for forgiveness. I never noticed this 'breadcrumbs thing' and now I find them everywhere. I'm a little embarrassed. However, it shows me once again what a good job you are doing. Thank you!
